R/functions_effects_DyNAM_choice.R

```diff
@@ -29,9 +29,8 @@ init_DyNAM_choice.default <- function(
   stats <- matrix(0, nrow = n1, ncol = n2) # check for poss
 
   # init a generic cache object
-  if ("cache" %in% .argsNames) {
-    cache <- stats
-  } else cache <- NULL
+  if ("cache" %in% .argsNames) cache <- stats
+  else cache <- NULL
```

Comment:
If we are keen to use `{rlang}` as a dependency, there is also rlang::%||%, which might be useful here.